### ISSUE TITLE
wallet_multiwallet test failed with BerkeleyBatch: Can't open database, duplicates fileid

### ISSUE THREAD

**Issue by MarcoFalke** [bitcoin/bitcoin#15733]
Error thrown once:

```
BerkeleyBatch: Can't open database w8_copy (duplicates fileid 595e23002f000000b5cefe1eff76000000000000 from w8)
```

In https://travis-ci.org/bitcoin/bitcoin/jobs/515252355#L4924

**Comment by promag** [comment]
That is expected:
https://github.com/bitcoin/bitcoin/blob/ba54342c9dd3f2e5cdeed9ac57f1924f0d885cc6/test/functional/wallet_multiwallet.py#L94-L97

Isn't the real error https://travis-ci.org/bitcoin/bitcoin/jobs/515252355#L4920 ?

**Comment by MarcoFalke** [comment]
@promag yes, seems so. This reminds me of the appveyor tests that fail when the `.cookie` does not have read permissions (yet?)

**Comment by promag** [comment]
Cookie file is created with a temporary name and then renamed to the final name (see `GenerateAuthCookie()`.

https://github.com/bitcoin/bitcoin/blob/35991b162fd142d76d83e672002846c8a40eb794/test/functional/test_framework/util.py#L326-L327

L327 fails but given the previous condition the file exists so I'd say we hit .
